curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65422ba173a18b1bedef1bb6.md

```diff
@@ -13,10 +13,10 @@ Inside the `if` block, call the `playSong()` function with the `id` of the first
 
 # --hints--
 
-You should create an `if` statement with the condition `userData?.currentSong === null`.
+You should create an `if` statement with the condition `userData?.currentSong`.
 
 ```js
-assert.match(code, /if\s*\(\s*userData\?\.currentSong\s*===\s*null\s*\)\s*\{/)
+assert.match(code, /if\s*\(\s*!\s*userData\?\.\s*currentSong\s*\)\s*\{/)
```

Comment:
same issue here.

We shouldn't force one answer here.
while `!userData?.currentSong` is valid so is the original answer of `userData?.currentSong === null`
The goal of this PR should be to allow for multiple valid answers instead of forcing campers to write just one fixed answers.

We want to test them to make sure they understand the concept.
So multiple valid answer should be allowed.

Please update the tests accordingly 👍